### DISCUSSION TITLE
File uploader Style page updates and some design considerations

### DISCUSSION THREAD

**Discussion by mbgower** [carbon-design-system/carbon#11206] [Help] [answered]
@laurenmrice As discussed at CAG, a number of the named elements on the File Uploader style page on the website do not reinforce the anatomy language on the Usage page. The first of these is "Label" which has been changed to 'Heading' in a [PR](https://github.com/carbon-design-system/carbon-website/pull/2868) However, many of the element names could better match the terms used on the Usage page. They are identified in a [commented pdf](https://ibm.box.com/s/ssd1dfnk9d5cydc5auvtp7ul98pj7jze).

As well, I'm wondering if the "x" is the best icon to use for the File delete mechanism. Doesn't Carbon normally use a trash can icon? That would seem to make more sense than "x" which tends to be used for "close" mechanisms.

**Comment by mbgower** [comment]
This discussion has been **resolved** through updates to the Style and Usage pages on Carbon by @laurenmrice.
@joshblack I don't see an ability to resolve this discussion. Are they deleted when resolved?

**Comment by laurenmrice** [comment, marked as answer]
The above content and image updates to the file uploader component have been made here. https://github.com/carbon-design-system/carbon-website/pull/2887
